doc/plugins/limit-req.md

```diff
@@ -76,7 +78,7 @@ Then add limit-req plugin:
 
 ![add plugin](../images/plugin/limit-req-2.png)
 
-## Test Plugin
+**Test Plugin**
```

Comment:
why change the doc style?

Reply:
Use `Test Plugin` as the title, and the title of "Test Plugin" in the following text, the repeated title, looks hard to understand. I think it is better after modification, do you think?